nginx/nginx_TLS.conf

```diff
@@ -113,6 +113,17 @@ http {
       # redirects, we set the Host: header above already.
       proxy_redirect off;
     }
+    # Used by Kubernetes liveness and readiness checks
+    location = /nginx_health {
+      return 200 "Born to be alive!\n";
+      access_log off;
+    }
+    location = /uwsgi_health {
+      rewrite /.+ /login?next=/ break;
+      include /run/defectdojo/uwsgi_pass;
+      include /etc/nginx/wsgi_params;
+      access_log off;
```

Comment:
can the application be normally accessed from /uwsgi_health endpoint, with access_log off? doesn't that have security implications like lack of logging in case of password brute-force using this endpoint?

Reply:
we could maybe use  some dummy django endpoint, other than login, that leads nowhere

Reply:
The best would be to have a real django readiness and liveness endpoint.

Since location use an = sign, it will only respond to /uwsgi_health, not /uwsgi_health/somestuff. And the rewrite will always redirect to /login?next=/. I'll had a condition to only forward if the method is GET. What do you think?

Reply:
is the `/login?next=/` logged in access log upon `/uwsgi_health` access ? If yes, no problem you can leave it, I just wasn't sure of how the rewrite would be handled.

Reply:
adding that GET limitation might be a good idea though just in case :) .

Reply:
just tried: it is currently possible to login through the /uwsgi_health uri, and we are missing some nginx access logs in this case: the ones for `GET /login?next=/` and `POST /login?next=/`

so it may be a way to bypass some IDS based on access logs. 

I think allowing only GET like you said should be enough

thanks!

Reply:
It's done!

Reply:
great!